### DISCUSSION TITLE
mmapping support

### DISCUSSION THREAD

**Discussion by Ar57m** [RWKV/rwkv.cpp#129] [General]
does rwkv support mmapping like on llama.cpp? if not, can you guys implement it(if possible)? it would be useful to not full the entire ram and crashing especially on Android(termux).

**Comment by saharNooby** [comment]
Hi! There is no `mmap` support, [the feature request is here](https://github.com/saharNooby/rwkv.cpp/issues/50).

**Reply by Ar57m** [reply]
thanks, It's been a while since there have been any updates in the feature request
